core/services/chainlink/application.go

```diff
@@ -197,6 +202,7 @@ func NewApplication(config *orm.Config, ethClient eth.Client, advisoryLocker pos
 		shutdownSignal:           shutdownSignal,
 		balanceMonitor:           balanceMonitor,
 		explorerClient:           explorerClient,
+		monitoringEndpoint:       monitoringEndpoint,
```

Comment:
Why keep a reference to it on the application?

Reply:
Had assumed it needed to be shutdown, it does not, removed.